karabo_data/reader.py

```diff
@@ -662,6 +662,51 @@ def get_array(self, source, key, extra_dims=None, roi=by_index[...]):
             sorted(non_empty, key=lambda a: a.coords['trainId'][0]), dim='trainId'
         )
 
+    def get_dask_array(self, source, key):
+        """Get a Dask array for the specified data field.
+
+        Dask is a system for lazy parallel computation. This method doesn't
+        actually load the data, but gives you an array-like object which you
+        can operate on. Dask loads the data and calculates results when you ask
+        it to, e.g. by calling a ``.compute()`` method.
+        See the Dask documentation for more details.
+
+        If your computation depends on reading lots of data, consider creating
+        a dask.distributed.Client before calling this.
```

Comment:
What about adding a recipe on how to use the client or even dask-jobqueue  for slurm integration to the docs?

Reply:
Yes, good point. I can turn the script I've been testing with into an example.